### ISSUE TITLE
Lump/spiltter argument aka Code Cleanup

### ISSUE THREAD

**Issue by valentinedwv** [hydroshare/hydroshare#52]
Are the standards lumping or splitting?
Presently seems to be lumping, but splitting is possible;
Examples. Models.py could be a folder with an init and models separated.

Why... can be hard to dive in?
I was looking at the my-resources page to look into searching.
 I looked for a my-resources class in hs_core, and a my-resources path in urls.py 

Finally did a search. It's tagged as a processor for the my-resources page is buried in the hs_core/views/__init__.py

@processor_for('my-resources')
def my_resources(request, page):

Never would have found it. There are other classes in the views folder.
Basically, having code for a common view in a directory with some classes split, and others buried in __init__.py seems to be bad practice.
